src/core/api/link_api.cpp

```diff
@@ -418,6 +429,11 @@ otError otLinkCslSetPeriod(otInstance *aInstance, uint16_t aPeriod)
     return error;
 }
 
+otError otLinkCslSetPeriodMs(otInstance *aInstance, uint32_t aPeriod)
+{
+    return otLinkCslSetPeriod(aInstance, uint16_t(aPeriod * 1000 / kUsPerTenSymbols));
```

Comment:
```suggestion
    return otLinkCslSetPeriod(aInstance, static_cast<uint16_t>(aPeriod * 1000 / kUsPerTenSymbols));
```
Use static_cast to be consistent with OT style.

Reply:
thx. `./script/make-pretty` did the job.